actions/annotations-guide.mdx

```diff
@@ -38,6 +38,110 @@ Bounding box is a rectangular box used to highlight the location of an object. O
 <Tip>
 For more, check out this video tutorial
 </Tip>
+---
+title: "Client Review Using Grid View"
+description: "Learn how clients can efficiently review and manage annotated files in Labellerr using the grid view, with quick accept or reject options."
+icon: grid-2
+---
+
+This feature allows clients to view files in a grid view, enabling users to quickly accept or reject files with just one click. Below, we provide step-by-step guidance on how to use this functionality.
+
+### 1. Access the Annotated file
+
+*   Labeling must be completed within the **Label** section.
+*   Once labeling is done, files should be submitted.
+*   Files must be accepted by a reviewer. All reviewer-accepted files will then appear in the **In Client Review** section.
+*   Next, navigate to the **Files** section by clicking on it.
+
+<Frame>
+  <img src="https://cdn.labellerr.com/1%20%20Documentation/08682f4f-2064-45b6-a740-8879155a3ee5.webp" alt="Access Files Section" />
+</Frame>
+
+*   At the top of the page, in the **File Filters** section, you'll find the status bar. Use the dropdown menu to select the '**Ready for Client Review**' filter to display relevant files. You may also apply other filters as needed to narrow down your search and click on the **Apply Filters**.
+
+<Frame>
+  <img src="https://cdn.labellerr.com/1%20%20Documentation/6fecebb1-d97d-4143-a629-c2b5060928e1.webp" alt="Apply Filters" />
+</Frame>
+
+Once the filter is applied, all your submitted files will be visible in the list, allowing you to view them easily in one place.
+
+<Frame>
+  <img src="https://cdn.labellerr.com/1%20%20Documentation/dfbd5356-a81c-4bfc-bf32-e3adc2cdb0e0.webp" alt="Filtered Files List" />
+</Frame>
+
+### 2. Review files in Grid View
+
+Once the files appear, hover over the icon on the top right bar to reveal the '**Grid View**' option, then click it to switch to grid view.
+
+<Frame>
+  <img src="https://cdn.labellerr.com/1%20%20Documentation/977fcd4d-c3bd-4115-bd77-dded2451e238.webp" alt="Switch to Grid View" />
+</Frame>
```

Comment:
<img width="1557" height="993" alt="image" src="https://github.com/user-attachments/assets/ed7770e8-faf8-4150-91eb-40c805cf8d16" />
has some parsing error, can you help to correct that, and after adding any changed, please reverify them with running 'mint dev' on your local system